bitnami/thanos/Chart.yaml

```diff
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/thanos
   - https://thanos.io
-version: 12.4.0
```

Comment:
Version 12.4.1 has already been released, could you please bump the chart version to `12.4.2`?

https://github.com/bitnami/charts/blob/main/bitnami/thanos/Chart.yaml

Reply:
@migruiz4 Thanks, updated to 12.4.2